packages/lesswrong/components/users/NewUserCompleteProfile.tsx

```diff
@@ -146,7 +161,9 @@ const NewUserCompleteProfile: React.FC<NewUserCompleteProfileProps> = ({ current
       
       {/* Facebook user with no email fix (very small % of users) */}
```

Comment:
@jpaddison3 Does this Facebook case apply to us? If not, then could we remove this code?